voluncheer/chatroom/views.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
+
+from chatroom.models import Room
+
+# To be changed after UI is designed.
+
+
+@login_required
+def chat_homepage_view(request):
+    return render(
+        request,
+        "chatroom/chat_homepage.html",
+        {
+            "rooms": Room.objects.all(),
+        },
+    )
+
+
+@login_required
+def room_view(request, room_name):
+    room, created = Room.objects.get_or_create(name=room_name)
```

Comment:
Since created is not used you can just discard it by assigning to an `_`:

```suggestion
room, _ = Room.objects.get_or_create(name=room_name)
```